code/modules/mob/living/basic/heretic/fire_shark.dm

```diff
@@ -1,5 +1,6 @@
 /mob/living/basic/heretic_summon/fire_shark
-	name = "fire shark"
+	name = "Fire Shark"
```

Comment:
Just making sure, does this end up getting formatted as `That's a Fire Shark` or `That's Fire Shark`?

Reply:
The latter, actually - in fact, the names of _every_ Heretic summon does this, which feels a little silly for most of them. Maybe I should lowercase them all instead?

Reply:
no just prefix improper

```suggestion
	name = "\improper Fire Shark"
```